src/app/personside/infotabs/ytelser/sykepenger/info/Sykepengertilfellet.tsx

```diff
@@ -0,0 +1,67 @@
+import DescriptionList, {
+    DescriptionListEntries,
+    fjernEntriesUtenVerdi
+} from '../../../../../../components/DescriptionList';
+import {
+    convertBoolTilJaNei,
+    datoEllerNull,
+    NOKellerNull,
+    periodeEllerNull
+} from '../../../../../../utils/stringFormatting';
+import * as React from 'react';
+import { Forsikring, Sykepenger, Sykmelding } from '../../../../../../models/ytelse/sykepenger';
+import YtelserInfoGruppe from '../../felles-styling/YtelserInfoGruppe';
+
+interface Props {
+    sykepenger: Sykepenger;
+}
+
+function getForsikringEntries(forsikring: Forsikring | null): DescriptionListEntries {
+    if (!forsikring) {
+        return {};
+    }
+    return {
+        Forsikring: NOKellerNull(forsikring.premiegrunnlag),
+        'Gyldig forsikring': convertBoolTilJaNei(forsikring.erGyldig),
+        Forsikringsperiode: periodeEllerNull(forsikring.forsikret)
+    };
+}
+
+function getYrkesskadeEntries(sykeMeldinger: Sykmelding[]): DescriptionListEntries {
+    const sisteSykemelding = sykeMeldinger[0];
+    if (!sisteSykemelding || !sisteSykemelding.gjelderYrkesskade) {
+        return {};
+    }
+    const yrkesSkade = sisteSykemelding.gjelderYrkesskade;
+    return {
+        Yrkesskade: yrkesSkade.yrkesskadeart,
+        Yrkesskadedato: datoEllerNull(yrkesSkade.skadet),
+        Vedtaksdato: datoEllerNull(yrkesSkade.vedtatt)
+    };
+}
+
+function Sykepengertilfellet({ sykepenger }: Props) {
+    const sykepengeTilfelletEntries: DescriptionListEntries = {
+        'Sykemeldt fra og med': datoEllerNull(sykepenger.sykmeldtFom),
+        'Forbrukte dager': sykepenger.forbrukteDager,
+        Maksdato: sykepenger.slutt,
```

Comment:
Vet jeg pirker veeeldig nå, men var det en grunn til at disse 1-ords entry-keyene ikke kunne ha fnutter rundt seg? Pretty klaga?

Reply:
Husker vi prata om det, husker bare ikke konklusjon :D

Reply:
pga prettier som overstyrer :-P

Reply:
eller som Dan Abramov ville sagt det: vi installerte prettier for å komme oss ut av denne debatten :-P